tests/test_parsley.py

```
@@ -1,97 +1,412 @@
+        bit_str.push(b'\x9A\xBC', 16)          # 16 bits of data
+        msg_sid, msg_data = parsley.parse_bitstring(bit_str)
+        
+        # First 29 bits of 0x123456789ABC = 0x02468ACF
+        # Remaining 19 bits = 0x9ABC
+        assert msg_sid == b'\x02F\x8a\xcf'
+        assert msg_data == b'\x00\x9a\xbc'
+        
+    def test_parse_bitstring_empty(self):
+        bit_str = BitString() #just an empty bitstring
+        try:
+            msg_sid, msg_data = parsley.parse_bitstring(bit_str)
```

Ruff (0.14.2)

Unpacked variable `msg_sid` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)